### PR TITLE
Xcode selection blog post

### PR DESCRIPTION
**Pull request by susinmotion**

### CODE REVIEW

**Comment by susinmotion** [comment]
This shouldn't be merged until the following Bazel release.

File: _posts/2020-02-06-xcode-selection.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+layout: posts
+title: Changes to Xcode selection in Bazel
+authors:
+  - susinmotion
+---
+
+One of our major projects in the last year has been improving build times for interactive iOS builds by adding support for the Google-internal remote execution service from Mac. Combined with [dynamic execution]({% post_url 2019-02-01-dynamic-spawn-scheduler %}), which helps mitigate deficiencies in our remote Mac executors by running some actions locally, we are now able to deliver the best possible clean and incremental build times.
```
Comment (aiuto) [inline _posts/2020-02-06-xcode-selection.md]: Do we have to say "google internal" or can you skip that and just talk about remote execution for mac?

File: _posts/2020-02-06-xcode-selection.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+Since the new `xcode_config` requires both dependencies to be set, we will never default to a remote-only Xcode version. We considered allowing a remote-only Xcode version in the absence of a locally available Xcode, but decided that preventing builds from failing cryptically if there was no Xcode to execute local actions was more important than providing the flexibility for dynamic execution to behave like a purely remote strategy.
+
+## Configuring dynamic execution
+The last piece is having Bazel execute actions in the right location based on the availability of the selected Xcode. If we’ve selected a local- or remote-only Xcode, either via `--xcode_version` or by accepting the default, we want to keep the dynamic scheduler from trying to execute Xcode-related actions on the other system.
```
Comment (aiuto) [inline _posts/2020-02-06-xcode-selection.md]: You might as well say "local-only or remote-only".   Factoring out the |only| is a minor savings for a probably readability hit.

File: _posts/2020-02-06-xcode-selection.md (hunk 1 above)
Comment (susinmotion) [reply _posts/2020-02-06-xcode-selection.md]: Haven't we already had Bazel Mac remote execution capabilities for awhile?

File: _posts/2020-02-06-xcode-selection.md (hunk 2 above)
Comment (susinmotion) [reply _posts/2020-02-06-xcode-selection.md]: Done.

File: _posts/2020-02-06-xcode-selection.md (hunk 1 above)
Comment (susinmotion) [reply _posts/2020-02-06-xcode-selection.md]: Changed first sentence to be Bazel-focused, and moved previous intro to "setting the scene"

**Pull request merged**